packages/twenty-shared/src/workflow/schemas/workflow-run-state-step-info-schema.ts

```diff
@@ -3,6 +3,14 @@ import { workflowRunStepStatusSchema } from './workflow-run-step-status-schema';
 
 export const workflowRunStateStepInfoSchema = z.object({
   result: z.any().optional(),
-  error: z.any().optional(),
+  error: z.string().optional(),
   status: workflowRunStepStatusSchema,
+  get history() {
```

Comment:
This is the recommended way to declare recursive types with Zod 4. See: https://zod.dev/v4?id=recursive-objects